src/components/buttons/ButtonGroup.tsx

```diff
@@ -41,8 +41,8 @@ export function ButtonGroup({
 }
 
 const StyledButtonGroup = styled.span`
+  display: inline-flex;
```

Comment:
NP: why change the order?

Reply:
I usually like to have the display value first and follow with the CSS properties that change it.

Reply:
like here, the align-items and gap are properties that are tied to display: inline-flex, and change the way the inline flex looks.

Reply:
ok